cmd/drone-docker/main.go

```diff
@@ -279,6 +281,26 @@ func main() {
 			Usage:  "ssh agent key to use",
 			EnvVar: "PLUGIN_SSH_AGENT_KEY",
 		},
+		cli.StringFlag{
+			Name:   "builder-name",
+			EnvVar: "PLUGIN_BUILDER_NAME",
```

Comment:
Do we have any default values for buildx including name and driver?

Reply:
No default name as it's possible that the name already exists. We rely on docker to give us the name if it's not set in env.

https://github.com/drone-plugins/drone-buildx/pull/2/files#diff-b68159b6168f099a639e8c6f96d75f23333ab0f0505ee22a98268748db17ae4cR187